htdocs/class/xoopsform/formbuttontray.php

```diff
@@ -101,10 +101,17 @@ public function render()
     {
         // onclick="this.form.elements.op.value=\'delfile\';
         $ret = '';
-        if ($this->_showDelete) {
-            $ret .= '<input type="submit" class="formbutton" name="delete" id="delete" value="' . _DELETE . '" onclick="this.form.elements.op.value=\'delete\'">&nbsp;';
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            if ($this->_showDelete) {
+                $ret .= '<input type="submit" class="btn btn-danger" name="delete" id="delete" value="' . _DELETE . '" onclick="this.form.elements.op.value=\'delete\'">&nbsp;';
+            }
+            $ret .= '<input type="button" value="' . _CANCEL . '" onClick="history.go(-1);return true;" />&nbsp;<input type="reset" class="btn btn-default"  name="reset"  id="reset" value="' . _RESET . '" />&nbsp;<input type="' . $this->getType() . '" class="btn btn-warning"  name="' . $this->getName() . '"  id="' . $this->getName() . '" value="' . $this->getValue() . '"' . $this->getExtra() . '  />';
+        } else {        
```

Comment:
__[phpmd]__
* The method render uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->